styles/styles.css

```diff
@@ -0,0 +1,112 @@
+body {
+  background-color: #FFFFFF;
+}
+
+.container {
+  width: 960px;
+  margin: 0 auto;
+  background-color: gray;
+}
+
+.main-header {
+  height: 130px;
+  background-color: green;
```

Comment:
Your header is missing a `position: relative` declaration; if I set the `.logo` to a `left: 0`, the logo is positioned outside of the header because the parent isn't containing it. See screenshot below:

![screen shot 2015-09-20 at 8 16 31 pm](https://cloud.githubusercontent.com/assets/651702/9986163/bc849400-5fd4-11e5-95ea-7f1f47df23c9.png)